.gitignore

```diff
@@ -6,3 +6,6 @@ logs/*.log
 coverage
 Gemfile.lock
 .byebug_history
+vendor/
+.bundle/
+.vscode/
```

Comment:
https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.editorconfig.erb#L13 ;)

Reply:
https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig ?

Reply:
We can add this, but it is outside the scope of the PR. The vscode stuff is only because I'm using VSCode right now and didn't want to commit my config.